front/src/applications/stdcm/components/StdcmForm/StdcmOperationalPoint.tsx

```diff
@@ -16,7 +16,8 @@ type StdcmOperationalPointProps = {
   disabled?: boolean;
 };
 
-type Option = StdcmPathStep['location'] & { label: string };
+type CIOption = StdcmPathStep['location'] & { label: string };
+type CHOption = { label: string; id: string; coordinates: [number, number] };
```

Comment:
```suggestion
type CHOption = { label: string; id: string; coordinates: GeoJsonPointValue };
```
Can't we use this type ?